backstage/templates/scaffolder/defaults/parameters/pagerduty.yaml

```diff
@@ -0,0 +1,54 @@
+title: Integrations (optional)
+properties:
+  createPagerduty:
+    title: Create PagerDuty Service?
+    description: |
```

Comment:
why are you pre-formatting the newlines here? nothing *particularly* against it, it's just inconsistent with the rest of the project